### DISCUSSION TITLE
benchmark per algo/coin?

### DISCUSSION THREAD

**Discussion by gurkburk76** [RainbowMiner/RainbowMiner#1681] [Q&A] [answered]
So it would seem the ETC (ethereum classic) has overtaken Ravencoin(RVN) / KawPow according to whattomine on my old shitty 570 4gb cards but i don't think RBM gets that so i though i'de re-bench that by removing files from /Stats/Miners/
and AMD*Ethash_HashRate.txt and hope it would re-bench those. 
It didn't.
Waht the best way going about doing this WITHOUT re-benching everything? 👯‍♂️ 

(I did look into pools that i have and (ETC) is listed there on zergpool as a coin so that's all good.)

worth noting is that it looks like the ETC was recently added to zergpool, like in the last 24h or something.
That might throw things off.

**Comment by gurkburk76** [comment]
Seems like that when i plug in the actual numbers i'm getting it does come out in favor of kawpow, so i guess i need to tweak my cards a bit :)

**Comment by gurkburk76** [comment]
So i've been a bit hesitant regarding flashing nwe bios on my cards, but now i'm sad i didn't before :)
from ~87mh/s to ~03mh/s on ether, not bad.
I guess i *will* have to re-benchmark now haha.

**Comment by RainbowMiner** [comment, marked as answer]
You did right, but with the wrong files:
ETC has it's own algorithm. It is "Etchash". So you will have to delete all files `AMD*Etchash_HashRate.txt`

> Waht the best way going about doing this WITHOUT re-benching everything?

Open the http://localhost:4000 (or http://machinenameorip:4000), then go to page "Benchmarks". You will get a sortable and searchable list of all benchmarks. Either click on the rebench-symbol to the right of an entry, or use the checkboxes to select multiple candidates and then click on the "Benchmark selected" button. For convenience, there is a check-all checkbox at the top of the table. So you could simply select "Etchash" with the dropdown, then click the check-all checkbox at the top of the table, then click "Benchmark selected". Now you would rebench Etchash on all miners.
